src/odemis/acq/test/calibration_test.py

```diff
@@ -175,7 +176,7 @@ def test_load_background(self):
         # Background data
         dcalib = numpy.array([1, 2, 2, 3, 4, 5, 4, 6, 9], dtype=numpy.uint16)
         dcalib.shape += (1, 1, 1, 1)
-        wl_calib = 400e-9 + numpy.array(range(dcalib.shape[0])) * 10e-9
+        wl_calib = 400e-9 + numpy.array(list(range(dcalib.shape[0]))) * 10e-9
```

Comment:
For numpy, you can write more directly:
`numpy.arange(dcalib.shape[0])`
(There are a couple of similar lines below)